subsys/bluetooth/audio/aics.c

```diff
@@ -67,34 +67,39 @@ static ssize_t read_description(struct bt_conn *conn,
 	BT_GATT_SECONDARY_SERVICE(BT_UUID_AICS),                               \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_STATE,                             \
 			       BT_GATT_CHRC_READ | BT_GATT_CHRC_NOTIFY,        \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY,   \
 			       read_aics_state, NULL, &_aics),                 \
 	BT_GATT_CCC(aics_state_cfg_changed,                                    \
-		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE_ENCRYPT),           \
+		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE_ENCRYPT                 \
+			 | BT_GATT_PERM_READ_128B_ENTROPY | BT_GATT_PERM_WRITE_128B_ENTROPY) \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_GAIN_SETTINGS,                     \
 			       BT_GATT_CHRC_READ,                              \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY    \
+				    | BT_GATT_PERM_WRITE_128B_ENTROPY,                   \
```

Comment:
`BT_UUID_AICS_GAIN_SETTINGS` is not writable

```suggestion
			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY,    \                 \
```

Reply:
Youre right, Ill fix